src/fluree/db/dbfunctions/fns.cljc

```diff
@@ -23,14 +23,23 @@
   "Coerces args that may be core async channels into values.
   Returns exception instead of args if any exception occurs during resolution."
   [args]
-  (go-try
-    (loop [[arg & r] args
-           acc []]
-      (if-not arg
-        acc
-        (if (channel? arg)
-          (recur r (conj acc (<? arg)))
-          (recur r (conj acc arg)))))))
+  ;; This fn looks more complicated than it needs to be at first glance, but other simpler forms
+  ;; I've tried run into go's fn boundary. Obviously something like `reduce` would since you'd be
+  ;; taking from the channel inside the anonymous fn arg. But even `for` exhibits this issue.
+  ;; Possibly because it macroexpands into an anon fn? Not sure. - WSM 2022-09-14
+  (go-try
+    ;; This loop is controlled by the arg count b/c valid args can be false, nil, etc. so
+    ;; you don't want to accidentally treat that as an "out of args" condition.
+    (let [arg-count (clojure.core/count args)]
```

Comment:
Th is is a .cljc file, I think your explicit clojure.core here could present issues. I know we use this other places in the file, so might already be living with this issue - or perhaps it is a non-issue.

Reply:
Yeah, I figured it was OK since it was already so prevalent. But I just did a quick test in a Node REPL:

```
❯ clj -M -m cljs.main --repl-env node
ClojureScript 1.11.60
cljs.user=> (clojure.core/+ 1 2 3)
6
```

Reply:
👍